### ISSUE TITLE
PLW3301 false positive if outer call has one argument and inner call has multiple arguments

### ISSUE THREAD

**Issue by dscorbett** [astral-sh/ruff#16163]
### Description

[`nested-min-max` (PLW3301)](https://docs.astral.sh/ruff/rules/nested-min-max/) has a false positive in Ruff 0.9.6 when the inner call is the outer call’s only argument and the inner call has multiple arguments.
```console
$ cat >plw3301.py <<'# EOF'
print(min(min([2, 3], [4, 1])))
print(max(max([2, 4], [3, 1])))
# EOF

$ python plw3301.py
2
3

$ ruff --isolated check --select PLW3301 plw3301.py --unsafe-fixes --fix
Found 2 errors (2 fixed, 0 remaining).

$ python plw3301.py
[2, 3]
[3, 1]
```
This is similar to #4682, in which the inner call has only one argument.

**Comment by vladNed** [comment]
yeah, seems like if formats the code thus modifying the behavior of the original code. 

I can start working on this if its fine

**Comment by vladNed** [comment]
found that the function just checks for arguments that are min/max inside the min/max without checking the types inside the nested one. 

Probably one improvement I can do is to check if inside there is a the nested one contains the same types as the parent one.

**Comment by zackw** [comment]
The converse is also a false positive:

```
x = max(floor, max(iterable))  # erroneous PLW3301
```

cannot be expressed any other way in general.
